UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Models/Response/CallBackResponse.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Newtonsoft.Json;
+
+namespace UKHO.ExchangeSetService.Common.Models.Response
+{
+    public class CallBackResponse
+    {
+        [JsonProperty("specversion")]
+        public string Specversion { get; set; }
```

Comment:
Should this be SpecVersion?

As in UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Configuration/EssCallBackConfiguration.cs

Reply:
As per the suggestion, i have made changes should this be SpecVersion.